Makefile

```diff
@@ -949,5 +955,5 @@ clean: clean-binaries ## Clean all make artifacts
 		libpod/pod_easyjson.go \
 		.install.goimports \
 		docs/build \
-		venv
+		.venv
```

Comment:
Good catch.

Reply:
It's setup at CI runtime with the `virtualenv .venv/blah...` commands in `setup_environment.sh`.  So it's arguably unnecessary (nothing in the Makefile creates it), I think Jhon's just being thorough.

Reply:
@TomSweeneyRedHat @cevich Correct, venv was the old name and removed, so I made sure the new name .venv was removed. The name change is more in alignment with tools like tox.